enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/CommonOtpService.java

```diff
@@ -68,21 +72,27 @@ public class CommonOtpService implements OtpService {
     public CommonOtpService(
             final OnboardingOtpRepository onboardingOtpRepository,
             final OnboardingProcessRepository onboardingProcessRepository,
+            final IdentityVerificationRepository identityVerificationRepository,
```

Comment:
I am not quite happy with separation of enrollment-server and onboarding server, but it is not target of this PR, may be improved later.

Reply:
Ok.